scripts/runtests.sh

```diff
@@ -163,7 +165,7 @@ echo "Cert gen/verify, CMS sign/verify, CA tests for all enabled OQS signature a
 
 # auto-detect all available signature algorithms:
 for alg in `$OPENSSL_APP list -signature-algorithms | grep oqsprovider | sed -e "s/ @ .*//g" | sed -e "s/^  //g"`
-do 
+do
```

Comment:
Ah, now I see your idea: This command _should_ fail to not return any PQ sig algs :-) Then indeed this can work :) Cute.